isobus/src/isobus_virtual_terminal_server_managed_working_set.cpp

```diff
@@ -7,19 +7,20 @@
 /// @copyright 2023 Adrian Del Grosso
 //================================================================================================
 #include "isobus/isobus/isobus_virtual_terminal_server_managed_working_set.hpp"
-
 #include "isobus/isobus/can_stack_logger.hpp"
 #include "isobus/utility/to_string.hpp"
 
+#include <cstring>
+
 namespace isobus
 {
 
 	VirtualTerminalServerManagedWorkingSet::VirtualTerminalServerManagedWorkingSet() :
+	  workingSetID(NULL_OBJECT_ID),
 	  objectPoolProcessingThread(nullptr),
 	  workingSetControlFunction(nullptr),
 	  processingState(ObjectPoolProcessingThreadState::None),
 	  workingSetMaintenanceMessageTimestamp_ms(0),
-	  workingSetID(NULL_OBJECT_ID),
```

Comment:
At some point we will probably want to change all the initializer lists to in-class initializers where possible. I was doing it the sort of old-school way when I wrote this. But it's fine for now.